ingestion_tools/scripts/importers/visualization_config.py

```diff
@@ -90,18 +90,21 @@ def _to_point_layer(
         name_prefix: str,
         color: str,
         resolution: tuple[float, float, float],
-        is_instance_segmentation: bool,
+        shape: str,
         **kwargs,
     ) -> dict[str, Any]:
-        return state_generator.generate_point_layer(
-            source_path,
-            f"{name_prefix} point",
-            self.config.https_prefix,
-            color=color,
-            scale=resolution,
-            is_visible=file_metadata.get("is_visualization_default"),
-            is_instance_segmentation=is_instance_segmentation,
-        )
+        args = {
+            "source": source_path,
+            "name": f"{name_prefix} point",
+            "url": self.config.https_prefix,
+            "color": color,
+            "scale": resolution,
+            "is_visible": file_metadata.get("is_visualization_default"),
+            "is_instance_segmentation": shape == "InstanceSegmentation",
```

Comment:
nit: not sure if it's just me, but this is kindof ugly imop.

Reply:
The variable name or doing the check against the shape value?

Reply:
setting the value for the `is_instance_segmentation` key with an inline boolean expression